package-lock.json

```diff
@@ -8975,17 +8970,17 @@
       "integrity": "sha1-pp8ObKWB4DkapXlBlw4XwwjdSGk=",
       "dev": true,
       "requires": {
-        "colors": "^1.3.0",
+        "colors": "latest",
         "connect": "3.4.x",
-        "cors": "^2.8.4",
-        "event-stream": "^3.3.4",
+        "cors": "latest",
+        "event-stream": "latest",
         "faye-websocket": "0.11.x",
         "http-auth": "2.4.x",
         "morgan": "^1.6.1",
-        "object-assign": "^4.1.1",
-        "opn": "^5.3.0",
-        "proxy-middleware": "^0.15.0",
-        "send": "^0.16.2",
+        "object-assign": "latest",
+        "opn": "latest",
+        "proxy-middleware": "latest",
+        "send": "latest",
```

Comment:
Why these changes? Do they fix some of the security warnings? If not, we should not include the changes in `package-lock.json` in our commits unless we add a new dependency.

Reply:
I just performed `npm-install`... I will revert these changes